receiver/dotnetdiagnosticsreceiver/dotnet/event_parser.go

```diff
@@ -0,0 +1,122 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dotnet
+
+import (
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/dotnetdiagnosticsreceiver/network"
+)
+
+// parseEventBlock parses an event block and returns a Metric slice containing
+// the raw representation of the metrics extracted from the event messages.
+// All of the previous parsing is for correct operation of the functionality
```

Comment:
What "previous parsing" does this refer to?

Reply:
Mostly `parseMetadataBlock`. I've updated the comment.